nixos/modules/services/x11/desktop-managers/gnome.nix

```diff
@@ -367,6 +366,9 @@ in
       services.gnome.tracker.enable = mkDefault true;
       services.hardware.bolt.enable = mkDefault true;
       services.packagekit.enable = mkDefault true;
+      services.pipewire.enable = mkDefault true;
+      services.pipewire.pulse.enable = mkDefault true;
```

Comment:
Looking at Nixpkgs some more, this will probably not interact with the ISO well:

https://github.com/NixOS/nixpkgs/blob/68bf774de230d7e8e3e5a51f303ea6576fd5e919/nixos/modules/installer/cd-dvd/installation-cd-graphical-base.nix#L36

There is also the following but that only seems to be used by the OVA, which is Plasma only at the moment:

https://github.com/NixOS/nixpkgs/blob/68bf774de230d7e8e3e5a51f303ea6576fd5e919/nixos/modules/profiles/graphical.nix#L17